error-analyzer/src/main/java/com/google/sps/servlets/Pagination.java

```diff
@@ -0,0 +1,151 @@
+// Copyright 2019 Google LLC
```

Comment:
Please re-iterate this code to make it more readable, modular and try to remove repeated pieces.
For ex. 
Create a function: getDisplayData(fileType) which returns display data.

Also add JUnit test for this.